asmcomp/arm/selection.ml

```diff
@@ -254,9 +254,9 @@ method private select_operation_softfp op args dbg =
       (Iintop_imm(Icomp(Iunsigned comp), 0),
        [Cop(Cextcall(func, typ_int, false, None), args, dbg)])
   (* Add coercions around loads and stores of 32-bit floats *)
-  | (Cload (Single, mut), args) ->
+  | (Cload {memory_chunk=Single; mutability=mut; is_atomic=false}, args) ->
```

Comment:
You can use field punning to avoid binding every field name with `=`. For example this could be written as 

```
| (Cload {memory_chunk=Single; mutability; is_atomic=false}, args) -> 
```